recipe/meta.yaml

```diff
@@ -22,7 +24,7 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python
+    - python >=3.6
```

Comment:
This constraint should not be added to packages that are not noarch.

Reply:
the constraints were modified accordingly.